src/arch/riscv/inc/arch/platform.h

```diff
@@ -10,6 +10,7 @@
 
 struct arch_platform {
     paddr_t plic_base;
+    paddr_t aclint_base;
```

Comment:
Since each aclint module can have completely independent bases, as per the spec, this should be `aclint_sswi_base`.